src/main/resources/templates/question-page.ftl

```diff
@@ -30,9 +29,9 @@
                         <a href="/question-list" float="right">View all the questions</a>
 		    			</div>
 		    			<!-- Here is the question body -->
-		    			<div class="vote-cell">
-    								<#include "./partials/vote_button.ftl" />
-    						</div>
+		    			<!-- <div class="vote-cell">
+								<#include "./partials/vote_button.ftl" />
```

Comment:
This <#include> is still interpreted when it's within a <!-- comment --> block. This causes the rest of the page to not be rendered correctly.